### DISCUSSION TITLE
Dropped Colour Proximity to an Arbitrary Colour

### DISCUSSION THREAD

**Discussion by Presently42** [Toinane/colorpicker#143] [Ideas]
I should like to propose a proposal: I'd like to have the ability to see how close to an arbitrary colour the colour which is dropped is.

To wit: I use the colour dropper. It gives me the HEX of a certain colour. In (perhaps the colour book?) another tab are saved colours. Colorpicker then indicates the closest of these pre-saved colours to the one currently in the dropper: perhaps the colour is 90 % Blue 150, for example - or 86 % Pink 175 - and so on.

**Comment by lulunac27a** [comment]
You can use the color difference based on RGB color space or Delta E methods, and then find the nearest color based on the list of colors.

**Reply by Presently42** [reply]
I seem to've found that exact option between the time of writing and now, as I've used this function many times indeed. That being said, I don't see an option to use Delta E; nor to specify which variant.

**Reply by Toinane** [reply]
Thank you for the update! I will try to implement this when possible

**Comment by mrAmrish** [comment]
Colorpicker 2.2.2 has some issue with them such as on **mac os Sequio** it does not get the color of for example on desktop if you go through picking a file color with  you cant get it,and everytime you click **_dropper_** it pops up **open setting**
![Screenshot 2024-11-20 at 11 38 13 AM](https://github.com/user-attachments/assets/d6141792-63a0-479b-8710-52632fde97e5)
